content/participate/review-changes.adoc

```diff
@@ -9,15 +9,15 @@ tags:
 ---
 
 Help us to review changes to code or documentation!
-All pull requests within Jenkins GitHub organizations are publicly open for reviews, and any reviews will be much appreciated.
+All pull requests within Jenkins GitHub organizations are publicly open for reviews and any reviews are much appreciated.
 
 == Code Reviews and Copy Editing
 
-Contributors and component maintainers may explicitly request request reviews from other community members.
-Below there are some queries which can help to find such open pull requests where help is needed:
+Contributors and component maintainers may explicitly request reviews from other community members.
+Below there are some queries which can help to find open pull requests where help is needed:
 
-- link:https://github.com/search?q=is%3Aopen+is%3Apr+team-review-requested%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing], mostly for Jenkins website
-- link:https://github.com/search?q=is%3Aopen+is%3Apr+%22jenkinsci%2Fcode-reviewers%22&type=Issues[Open pull requests for code reviews] (https://github.com/orgs/jenkinsci/teams/code-reviewers[@jenkinsci/code-reviewers] team is mentioned)
+- link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+is%3Aopen+is%3Apr+team-review-requested%3Ajenkins-infra%2Fcopy-editors[Open pull requests for copy editing], mostly for Jenkins website
+- link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+is%3Aopen+is%3Apr+%22jenkinsci%2Fcode-reviewers%22&type=Issues[Open pull requests for code reviews] (https://github.com/orgs/jenkinsci/teams/code-reviewers[@jenkinsci/code-reviewers] team is mentioned)
 - link:https://github.com/search?q=is%3Aopen+is%3Apr+%22jenkinsci%core-pr-reviewers%22&type=Issues[Open pull requests for code reviews in the Jenkins Core] (https://github.com/orgs/jenkinsci/teams/core-pr-reviewers[@jenkinsci/core-pr-reviewers] team is mentioned)
```

Comment:
I think the URL in this search needs to be:

https://github.com/search?q=org%3Ajenkinsci+is%3Aopen+is%3Apr+is%3Apr+team-review-core-pr-reviewers